src/services/node/node.jsx

```diff
@@ -0,0 +1,9 @@
+const EXPENSES_URL = 'https://budget-app-aa.herokuapp.com/expenses'
```

Comment:
to do configa jakiegoś z urlami endpointów w nim tylko `/expenses` `/categories` trzymajmy a url do api w zmiennej środowiskowej

Reply:
spoczko, ale to dopiero po z mergowaniu zmian